sdk/ai/ai-projects/samples-dev/agents/agents_basics_tracing_console.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Are these samples auto-generated or hand authored? The file naming convention doesn't line up with other samples in our repo

Reply:
They are had authored, I'll rename it to be camelCased.